site_scons/site_tools/roc/pkgconfig.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+def FindPkgConfig(env, toolchain):
+    if env.HasArg('PKG_CONFIG'):
+        return True
+
+    if toolchain:
```

Comment:
Let's add a comment with this link: https://autotools.io/pkgconfig/cross-compiling.html